### ISSUE TITLE
formio-utils is deprecated

### ISSUE THREAD

**Issue by dylanized** [formio/formio-service#11]
This library uses a module formio-utils which appears to be deprecated?

**Comment by travist** [comment]
Yep, I see that. We need to just replace references to that with ```formiojs/utils```. I can work on that change soon.
